scopes/react/react/webpack/webpack.config.base.ts

```diff
@@ -99,77 +84,44 @@ export default function (isEnvProduction = false): Configuration {
           // match the requirements. When no loader matches it will fall
           // back to the "file" loader at the end of the loader list.
           oneOf: [
-            // "postcss" loader applies autoprefixer to our CSS.
-            // "css" loader resolves paths in CSS and adds assets as dependencies.
-            // "style" loader turns CSS into JS modules that inject <style> tags.
-            // In production, we use MiniCSSExtractPlugin to extract that CSS
-            // to a file, but in development "style" loader enables hot editing
-            // of CSS.
-            // By default we support CSS Modules with the extension .module.css
+            // load styles, including scss, less, and
+            // * no need for sourceMaps - loaders respect `devTools` (preferably, 'inline-source-map')
+            // * miniCssExtractPlugin - (somehow) breaks hot reloading, so using style-loader in dev mode
             {
-              test: stylesRegexps.cssNoModulesRegex,
-              use: generateStyleLoaders(
-                merge({}, baseStyleLoadersOptions, {
-                  cssLoaderOpts: {
-                    importLoaders: 1,
-                    sourceMap: isEnvProduction || shouldUseSourceMap,
-                  },
-                })
-              ),
-              // Don't consider CSS imports dead code even if the
-              // containing package claims to have no side effects.
-              // Remove this when webpack adds a warning or an error for this.
-              // See https://github.com/webpack/webpack/issues/6571
+              test: stylesRegexps.allCssregex,
```

Comment:
I thought of extracting this loader array to own component, but we need to `require.resolve()` each loader from here anyways (unless we move them to the component.json of the new component?).

It's also likely we'd like to fine tune each loader options, so it makes sense to me to just leave it here.